OSD-2022.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: OSD-2022
+title: Open Source Day 2022
+permalink: /OSD-2022/
+---
+:page-liquid:
+
+
+Welcome Open Source Day participants! We're glad to have you join us for the hackathon.
+
+To get started, follow the steps below:
+
+. *Setup the necessary tools to get started.* We use Java 11, Maven, and GitHub for development. Follow the setup instructions for your operating system:
++
+++++
+{% include OSD-2022-logo-bullets.html %}
+++++
+
+. *Pick your https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#Java[IDE].* If it's your first time, we recommend an IDE which includes all the tools you'll need, like https://www.jetbrains.com/idea/[IntelliJ IDEA] or https://www.eclipse.org/ide/[Eclipse]. For those more familiar with Java development and using Maven, feel free to use whatever you'd like.
+. *Get familiar with WildFly.* Our link:../getting-started-for-developers/[Getting Started] page has some resources to help you learn more about the project. You don't need to go through all of the links, so here are a few sections to look at:
+    ** link:../getting-started-for-developers/#a-quick-introduction-to-the-project[Wild**Fly** Project Introductions]
+    ** link:../getting-started-for-developers/#getting-familiar-with-elytron[Getting familiar with Elytron]
+    ** link:../getting-started-for-developers/#getting-your-developer-environment-set-up[Building and testsuite comamnds]
+. *Start contributing!* Head over to GitHub to get started. The file https://github.com/wildfly-security/wildfly-elytron/blob/1.x/CONTRIBUTING.md[CONTRIBUTING.md] has everything you need to get started, including how to use our issue tracker (https://issues.redhat.com/issues/?filter=12383825[link]), open a pull request, and get your changes into Elytron!
```

Comment:
The good first issues link from our Elytron contribution guide is slightly different than what we're going to use for OSD.

Maybe we can reword to:

*Start contributing!* Whether you're new to security, new to Java, or new to tools like Git or Maven, we have a bunch of issues that are perfect for getting started on our project, no prior experience is needed. Check out our list of (https://issues.redhat.com/issues/?filter=12364234)[good first issues]. Once you have selected an issue you'd like to work on, make sure it's not already assigned to someone else. To assign an issue to yourself, simply click on "Start Progress". This will automatically assign the issue to you.

Our (https://github.com/wildfly-security/wildfly-elytron/blob/1.x/CONTRIBUTING.md#contributing-guidelines)[contribution guide] has all the information you need on how to format your PRs.

Reply:
Let's making "Start contributing" #3 in the list (just so people don't feel that step 3 is a pre-req for step 4).

And then maybe after the message us on slack line, we can add a new paragraph that starts with:

To get more familiar with WildFly Elytron, ...our Getting Started page....

WDYT?

Reply:
sounds good